<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTabWidget" name="LeftTabs">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="RightTabs">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTF">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="ActionPreferences"/>
   </widget>
   <addaction name="menuTF"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTest_item">
   <property name="text">
    <string>test item</string>
   </property>
  </action>
  <action name="ActionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
